units/Cargo.toml

```diff
@@ -23,6 +23,7 @@ internals = { package = "bitcoin-internals", version = "0.3.0" }
 serde = { version = "1.0.103", default-features = false, features = ["derive"], optional = true }
 
 [dev-dependencies]
+internals = { package = "bitcoin-internals", version = "0.3.0", features = ["test-serde"] }
 serde_test = "1.0"
 serde_json = "1.0"
```

Comment:
Are these still needed?

Reply:
Yeah they are, I only added them to crates that call the macro.